cmake/ports/glad/portfile.cmake

```diff
@@ -2,13 +2,21 @@ include(vcpkg_common_functions)
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 file(READ "${VCPKG_ROOT_DIR}/_env/EXTERNAL_BUILD_ASSETS.txt" EXTERNAL_BUILD_ASSETS)
+file(READ "${VCPKG_ROOT_DIR}/_env/USE_GLES.txt" USE_GLES)
 
 if (ANDROID)
     vcpkg_download_distfile(
         SOURCE_ARCHIVE
         URLS ${EXTERNAL_BUILD_ASSETS}/dependencies/glad/glad32es.zip
         SHA512 2e02ac633eed8f2ba2adbf96ea85d08998f48dd2e9ec9a88ec3c25f48eaf1405371d258066327c783772fcb3793bdb82bd7375fdabb2ba5e2ce0835468b17f65
     )
+elseif (USE_GLES)
+    vcpkg_download_distfile(
+        SOURCE_ARCHIVE
+        URLS ${EXTERNAL_BUILD_ASSETS}/dependencies/glad/glad32es.zip
+        SHA512 2e02ac633eed8f2ba2adbf96ea85d08998f48dd2e9ec9a88ec3c25f48eaf1405371d258066327c783772fcb3793bdb82bd7375fdabb2ba5e2ce0835468b17f65
+        FILENAME glad32es.zip
+    )
```

Comment:
Can this be merged into the `if()` above? They appear to be the same apart from one having a line `FILENAME` line.

Reply:
Apparently the GHA android builds fail if FILENAME glad32es.zip is set. (look at the commit history)
I am too unfamiliar with how the android builds work to fix that.